lib/css/zocial.less

```diff
@@ -240,9 +242,8 @@ button:-moz-focus-inner {
-    url('https://cdn.auth0.com/w2/font/zocial-regular-webfont.ttf') format('truetype'),
-    url('https://cdn.auth0.com/w2/font/zocial-regular-webfont.svg#zocialregular') format('svg');
```

Comment:
Mostly because they are outdated. All browsers except ie8 support woff. I also added the latest .ttf in base64 as a backup. Should be fine.